### PR TITLE
U50 XRT driver changes

### PR DESCRIPTION
**Pull request by mfcpickett**
Added 6 new sensors for U50

### CODE REVIEW

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c [hunk 1]
```diff
@@ -79,6 +79,13 @@
 #define	XMC_SNSR_CHKSUM_REG		0x1A4
 #define	XMC_SNSR_FLAGS_REG		0x1A8
 #define	XMC_HBM_TEMP_REG		0x260
+#define	XMC_VCC3V3_REG			0x26C
+#define	XMC_3V3_PEX_I_REG		0x278
+#define	XMC_VCC0V85_I_REG		0x284
+#define	XMC_HBM_1V2_REG			0x290
+#define	XMC_VPP2V5_REG			0x29C
+#define	XMC_VCCINT_BRAM_REG		0x2A8
+#define	XMC_HBM_TEMP2_REG		0x2B4
```
Comment (maxzhen) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c]: can we have a better name? HBM_TEMP_INTERNAL?

**Comment by maxzhen** [comment]
ok to test

**Comment by abalasa** [comment]
retest this please

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c (hunk 1 above)
Comment (mfcpickett) [reply src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c]: HBM_TEMP2 is the name of the register exposed at that address by CMC. Name changes should be directed at the CMC developers.

**Comment by maxzhen** [comment]
Actually, it's better to just use the power sysfs node exported from cmc sub-device driver and not doing the power calculation in xbutil. I'll merge this PR. Then @chienwei-lan can make the change.

**Pull request merged**